src/patternfly/demos/Page/page-template-header-tools-elements.hbs

```diff
@@ -1,36 +1,19 @@
 {{#> page-header-tools}}
-  {{#> page-header-tools-group page-header-tools-group--modifier="pf-m-hidden pf-m-visible-on-lg"}}
-    {{#> page-header-tools-item}}
+  {{#> page-header-tools-group}}
+    {{> page-template-header-tools-notification-badge page-template-header-tools-notification-badge--item--modifier="pf-m-hidden pf-m-visible-on-lg"}}
```

Comment:
I don't think I see the notification bell?  https://patternfly-pr-3115.surge.sh/documentation/core/demos/page/default-nav

Reply:
Oh I removed the notification bell from the default demos, they're only on the drawer demos now.